docs/src/main/asciidoc/_admin-guide/subsystem-configuration/MicroProfile_OpenAPI.adoc

```diff
@@ -90,3 +90,5 @@ The Eclipse MicroProfile OpenAPI implementation is provided by the SmallRye Open
 * http://github.com/smallrye/smallrye-open-api/[SmallRye OpenAPI]
 
 ****
+
+NOTE: References in this document to Java API for RESTful Web Services(JAX-RS) refer to Jakarta RESTful Web Services unless otherwise noted.
```

Comment:
JAX-RS appears five more times in this file.

Reply:
I'm following this document - https://jakarta.ee/legal/acronym_guidelines/ 
where it has mentioned -

> If a specification document (or related materials) contains references to an Oracle/JCP acronym, include a sentence such as the following: “References in this document to JMS refer to the Jakarta Message Service unless otherwise noted." The sentence needs to be a legible size and should appear early in the specification document (or related materials), not in an illegibly small footnote. A reader should not have to scroll, turn a page, or click a link to get to the sentence.

That's why I have mentioned the note at the end of the file where a many times repeated replacement of 'JAX-RS' with the much more verbose 'Jakarta RESTful Web Services' is pretty clunky.